src/sonic-eventd/Makefile

```diff
@@ -0,0 +1,83 @@
+RM := rm -rf
```

Comment:
The other components in this repo is using Debian package build infra, not plain Makefile. Could you adapt?

Reply:
This is using debian package -- pretty similar to dhcp-relay